testscripts/shellenv/node/devbox.lock

```diff
@@ -0,0 +1,90 @@
+{
```

Comment:
is it necessary to add this file? currently, we have devbox.json adding `nodejs@18` and so upon execution we download the latest (and secure) version of it. With the lock file, we'll need to do more work to maintain and update it